grails-app/assets/javascripts/spApp/controller/addAreaCtrl.js

```diff
@@ -130,16 +130,14 @@
                                         }
                                 },
                                 //Error
-                                function(response){
-                                    if (response.status == 403 || response.status == 401) {
-                                        bootbox.alert($(i18n(539,"Authentication failed or login session expired, Please login again!")))
-                                    } else if (response.data.error) {
-                                        bootbox.alert("Error:" + response.data.error);
-                                    } else {
-                                        bootbox.alert("Unexpected error. Please check logs for more information")
-                                        console.log(response.data)
+                                function(error) {
+                                    if (!error.handled) {
```

Comment:
I can't see where `error.handled` is being set...
Also, where are 403 and 401 checks now being looked for? Its not clear from the code change....

Reply:
There is an Http interceptor in spApp.js

https://github.com/AtlasOfLivingAustralia/spatial-hub/blob/392_facet_query/grails-app/assets/javascripts/spApp/spApp.js#L114